### DISCUSSION TITLE
Updatecycle

### DISCUSSION THREAD

**Discussion by dominikjas** [tobybatch/kimai2#523] [Q&A]
Hello,

we are using your kimai docker image for our customers and we are a bit curious, when you will probably update to latest repo of the general kimai image https://github.com/kimai/kimai/releases -> 2.0.29

Thank you!

**Comment by kevinpapst** [comment]
> for our customers

Time for a donation 👍

**Comment by kevinpapst** [comment]
https://github.com/tobybatch/kimai2/discussions/524
